src/D2L.CodeStyle.Analyzers/Properties/AssemblyInfo.cs

```diff
@@ -10,7 +10,7 @@
 
 [assembly: ComVisible( false )]
 
-[assembly: AssemblyVersion( "0.9.0.0" )]
-[assembly: AssemblyFileVersion( "0.9.0.0" )]
+[assembly: AssemblyVersion( "0.9.1.0" )]

```

Comment:
That is odd - I pulled this repo for the first time today, and it had it at 0.9.0.0 locally prior to my work... I'll bump it again to 0.9.2.0